hashes/sha256/hash.zok

```diff
@@ -0,0 +1,7 @@
+import "hashes/sha256/sha256" as sha256;
```

Comment:
compiler error: wrong sha256 function used => hashes/sha256/512bitPadded is correct for these inputs